songbird/parameter_info.py

```diff
@@ -60,4 +64,5 @@
     "checkpoint-interval": 3600,
     "summary-interval": 10,
     "summary-dir": "summarydir",
+    "random-seed": 0,
```

Comment:
```suggestion
    "random-seed": None,
```
I'm still leaning towards having the default be None, otherwise the option to not specify a random seed will no longer exist. For example, if you have a multiple random runs, you do things like take averages, or take the best fit amongst multiple random runs.  If this option is not available, it won't be possible for the user to not specify a random seed.

Reply:
Thanks! And I appreciate the feedback on the PR.

I think it is important to mandate a seed for Q2. For a software that has a mission of reproducibility, you should not be able to create artifacts that you cannot reproduce. If someone were to give me an artifact created by songbird with seed set to None, I’d never be able to reproduce it exactly.

I’m also not sure why you can’t achieve your use case by setting the random seed differently, multiple times? This use case is actually the reason I had two different seeds originally, so you could keep the same train/test split, but change the model fitting.

Also setting a None seed is still available via the CLI, there is a unit test for it in this PR.